dapps/README.md

```diff
@@ -5,7 +5,7 @@
 |   1 | ACE                                  | https://ace.web3go.xyz/#/                            | utilities                     |
 |   2 | Acala App (Desktop mode)             | https://apps.acala.network/                          | bridge,dex,staking            |
 |   3 | ArthSwap                             | https://app.arthswap.org/#/swap                      | dex,staking,evm               |
-|   4 | Astar DApp Hub                       | https://portal.astar.network/                        | bridge,staking,evm            |
```

Comment:
changes in README.md are done by script, so I would prefer to leave it unchanged